script.js

```diff
@@ -0,0 +1,193 @@
+// Home -work for lesson #13 //
+
+let country = "Moldova";
+let continent = "Europe";
+let population = 2.615;
+
+console.log("Country name is " + country);
+console.log(country + "'s continent is " + continent);
+console.log(country + "'s population is " + population + " billions");
+
+let isIsland = false;
+
+let coutryDescription = country + " is located in " + continent + ", " + population + " mln people live here";
```

Comment:
You had to use Template Literals here

Reply:
Technically the solution is correct. 
But agree with Iuliana,  would be more useful to use template literals